### PR TITLE
Join table visits

### PR DESCRIPTION
**Pull request by martinkarugaba**
# :recycle: Changes made

- [ ] Created a table named vets.
- [ ] Created a "join table" called specializations
- [ ] Created a "join table" called visits
- [ ] Inserted data for vets.
- [ ] Inserted data for specializations.
- [ ] Inserted data for visits.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -103,4 +103,143 @@ FROM animals
...
+SELECT species.name AS specialty
+FROM specializations
+JOIN vets ON specializations.vet_id = vets.id
+JOIN visits ON vets.id = visits.vet_id
+JOIN animals ON visits.animal_id = animals.id
+JOIN species ON specializations.species_id = species.id
+WHERE vets.name = 'Maisy Smith'
+GROUP BY species.name
+ORDER BY COUNT(visits.vet_id) DESC
+LIMIT 1;
```
Comment (muneebulrehman) [inline queries.sql]: As per the screenshot you have attached for this query, it is returning empty results which is wrong. As per the data, it should return **Digimon**. Kindly fix this issue.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @martinkarugaba , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive PR
- Good docs
- Data inserted correctly.

_Note_: Kindly remember to update your screenshots once you have fixed your query.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (martinkarugaba) [reply queries.sql]: # :recycle: Changes made
- [ ] Fix last query
last-query

**Review by tugsem — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️
Hello  @martinkarugaba 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

 ## Highlights 🚀
✔️Good job on fixing the screenshot for the query.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by martinkarugaba** [comment]
Hi @tugsem, you have not told me where the issue with my current submission is. Please point to where the issue might be.

**Review by AshakaE — APPROVED** [review]
Hi @martinkarugaba   , 

### Highlights ✨ 

- Thanks for making the requested changes 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tugsem** [comment]
> Hi @tugsem, you have not told me where the issue with my current submission is. Please point to where the issue might be.

Sorry @martinkarugaba I think there was a problem because I remember I have written the requested change. Sorry for it. I was going to say please update the query which you already did later.  Have a nice day!

**Review by V-Blaze — COMMENTED** [review]
Hi @martinkarugaba  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy


## Highlights
- Queries written correctly ✔✔
- Clean and well-organized codebase :heavy_check_mark: 
- Querry result screenshots added to pull request :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**